.pre-commit-config.yaml

```diff
@@ -5,7 +5,11 @@ repos:
     hooks:
       - id: trailing-whitespace
       - id: check-added-large-files
-        exclude: src/dashboard/trulens/dashboard/.*/dist/.*
+        exclude: |
+          (?x)^(
+            src/dashboard/trulens/dashboard/.*/dist/.*|
```

Comment:
I'm not sure what this is for. I don't even see anything that satisfies this glob.